zer0-os/zOS
# Move the flow of logging in fully into sagas

### What does this do?

Moves the login, page load, change account flows fully into sagas.

### Why are we making this change?

Takes away a lot of the logic that was happening based on component property changes. Now things are happening based on actual user flows or in reaction to user/authentication events.

### How do I test this?

As an email user and web3 user:
* Register for a new account
* Login from scratch
* Reload the page when logged in
* Reload the page when not logged in

In all cases verify that chats are loaded, the page is in the right opening state, etc.
